app-tasks/dags/sentinel2/import_sentinel2_scenes.py

```diff
@@ -1,60 +1,29 @@
 import os
-import logging
 
-from airflow.operators.python_operator import PythonOperator
+from airflow.operators.bash_operator import BashOperator
 from airflow.models import DAG
 from datetime import datetime, timedelta
 
-from rf.uploads.sentinel2 import create_sentinel2_scenes
-from rf.utils.exception_reporting import wrap_rollbar
-
-rf_logger = logging.getLogger('rf')
-ch = logging.StreamHandler()
-ch.setLevel(logging.INFO)
-formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-ch.setFormatter(formatter)
-rf_logger.addHandler(ch)
-
-logger = logging.getLogger(__name__)
-
-
 seven_days_ago = datetime.combine(
     datetime.today() - timedelta(7), datetime.min.time())
 
-
 args = {
     'owner': 'raster-foundry',
     'start_date': seven_days_ago
 }
 
-
 dag = DAG(
     dag_id='import_sentinel2_scenes',
     default_args=args,
     schedule_interval=None,
     concurrency=int(os.getenv('AIRFLOW_DAG_CONCURRENCY', 24))
 )
 
+bash_cmd = "java -cp /opt/raster-foundry/jars/rf-batch.jar com.azavea.rf.batch.Main import_sentinel2 {}".format(seven_days_ago.strftime('%Y-%m-%d'))
```

Comment:
I think this should be like the Landsat 8 importer with `{{ yesterday_ds }}` right?